### PR TITLE
use https for License to ensure others do too

### PR DESCRIPTION
**Pull request by gdi2290**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -314,6 +314,6 @@ If you want to create your own list, please read the [create list instructions](
 
 ## License
 
-[![CC0](http://i.creativecommons.org/p/zero/1.0/88x31.png)](http://creativecommons.org/publicdomain/zero/1.0/)
+[![CC0](https://i.creativecommons.org/p/zero/1.0/88x31.png)](http://creativecommons.org/publicdomain/zero/1.0/)
```
Comment (sindresorhus) [inline readme.md]: Can you make the other URL https too?

**Comment by gdi2290** [comment]
@sindresorhus .done()

**Comment by arthurvr** [comment]
Thanks :)

**Comment by gdi2290** [comment]
awesome :)

**Pull request merged**